geemusic/intents/playback.py

```diff
@@ -81,6 +82,25 @@ def prev_song():
 
         return audio().play(stream_url)
 
+@ask.intent("AMAZON.ShuffleOnIntent")
+def shuffle_tracks():
+    shuffled_songs = queue.song_ids
+    
+    if shuffled_songs == [] or None:
+        return statement("There are no songs to shuffle.")
+    else:
+        api = GMusicWrapper.generate_api()
+    
+        # This will randomly reorder the song ids thereby "shuffling" the tracks
+        random.shuffle(shuffled_songs)
```

Comment:
I'd prefer to use `random.sample(song_ids, len(song_ids)` in order to prevent any in-place modification of data structures.